docs/how-to/ownership.mdx

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 7
+title: Modeling Ownership
+---
+
+Ownership plays an important role in managing and securing access to resources - this guide explains how to setup ownership using Permit
+
+## When Do We Need Ownership?
+
+Ownership becomes crucial in scenarios where control over data, files, or any digital assets is essential to ensure that only authorized individuals or entities can access, modify, or share them.
```

Comment:
I feel like this question isn't correct because ownership is kinda basic of permissions.
I would remove the title and maybe put an intro paragraph such as the following

```suggestion
---

Ownership is the first building block of fine-grained authorization as we always want first to ensure data that owned by user is accessible only by them and by others that privileged by the owner.
```

Reply:
Done